docs/DataModels/DevLakeDomainLayerSchema.md

```diff
@@ -90,6 +90,7 @@ An `issue` is the abstraction of Jira/Github/GitLab/TAPD/... issues.
 | `updated_date`              | datetime | 3          | The last time issue gets updated                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |         |
 | `resolution_date`           | datetime | 3          | The time the issue changes to 'DONE'.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |         |
 | `lead_time_minutes`         | int      |            | Describes the cycle time from issue creation to issue resolution.<ul><li>For issues whose type = 'REQUIREMENT' and status = 'DONE', lead_time_minutes = resolution_date - created_date. The unit is minute.</li><li>For issues whose type != 'REQUIREMENT' or status != 'DONE', lead_time_minutes is null</li></ul>                                                                                                                                                                                                                                                                                      |         |
+| `original_project`          | varchar  | 255        | The name of project                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |         |

```

Comment:
Text change:
The name of the original project. Transformed from a Jira project's name, a TAPD workspace's name, etc.